agent/pkg/handler/linkerd_info.go

```diff
@@ -57,6 +58,31 @@ func (h *LinkerdInfo) Stop() {
 	close(h.stopCh)
 }
 
+func (h *LinkerdInfo) handleAuthInfo(ctx context.Context) {
+	servers, err := h.k8s.GetServers(ctx)
+	if err != nil {
+		h.log.Errorf("error getting servers: %s", err)
```

Comment:
if i run this with a pre-2.11 linkerd, we'll log an error every minute:
```
ERRO[2021-09-27T18:36:12Z] error getting servers: the server could not find the requested resource  handler=linkerd_info
```
is there a way to check whether the CRD is installed, or, maybe just log a warning depending on what error comes back?

Reply:
I think you can do something similar to https://github.com/linkerd/linkerd2/blob/main/pkg/k8s/authz.go#L96 to check if a CRD is present in the cluster.